week-5.md

```diff
@@ -0,0 +1,49 @@
+
+# Week Five Info
+
+> "Our third objective of smart contract design is privity, the principle that knowledge and control over the contents and performance of a contract should be distributed among parties only as much as is necessary for the performance of that contract. This is a generalization of the common law principle of contract privity, which states that third parties, other than the designated adjudicators and intermediaries, should have no say in the enforcement of a contract." - Nick Szabo, ["Formalizing and Securing Relationships on Public Networks"](https://firstmonday.org/ojs/index.php/fm/article/view/548/469-publisher=First) 1997-09-01
```

Comment:
nit: can you italicize _privity_ (as it is in the referenced article), since it's probably a new term for many readers.

Reply:
yep